pytest_fixtures/core/sys.py

```diff
@@ -67,6 +67,17 @@ def install_cockpit_plugin(session_target_sat):
     setting_object.update({'value'})
 
 
+@pytest.fixture(scope='module')
+def enable_capsule_for_registration(module_target_sat):
+    """Enable registration and template features for capsule required for
```

Comment:
I see the fixture is being used in `test_global_registration_with_capsule_host` for instance. This particular test registers the host to the external capsule. How does the installer that is run on Satellite impacts the external capsule?

Reply:
yes, you are right, missed that there were also tests for external capsules, I will check it and remove the decorator from external ones because they seem to work without any changes. Thank you